src/Interpreters/InterpreterShowProcesslistQuery.cpp

```diff
@@ -12,7 +12,7 @@ namespace DB
 
 BlockIO InterpreterShowProcesslistQuery::execute()
 {
-    return executeQuery("SELECT * FROM system.processes", getContext(), true);
+    return executeQuery("SELECT * FROM system.processes ORDER BY query_id", getContext(), true);
```

Comment:
I propose to use elapsed_ms DESC

Reply:
I was also not certain what the best sort key is. Yes, for usage in ` watch -n1 "clickhouse-client --query='SHOW PROCESSLIST'"` (which is promoted in the [docs](https://clickhouse.com/docs/en/sql-reference/statements/show#show-processlist)) sorting by descending elapsed time is better.